knotx-core/build.gradle.kts

```diff
@@ -0,0 +1,137 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+import org.nosphere.apache.rat.RatTask
+import java.util.Date
+
+/*
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id("java-library")
+  id("maven-publish")
+  id("org.nosphere.apache.rat") version "0.4.0"
+}
+
+description = "Knot.x Core"
+
+dependencies {
```

Comment:
You could probably move `deps` into separate file and just apply them here - actual build script would be more readable.

Reply:
fixed.